Error marshalling fails on 32-bit systems

I'm running a [test build](https://koji.fedoraproject.org/koji/taskinfo?taskID=32730003) across all arches using 0.9.2, and it fails 3 tests on 32-bit systems (i686 and armv7hl):
```
--- FAIL: TestUnmarshalError (0.00s)
    encoding_test.go:78: got:  wanted: error message
--- FAIL: TestMarshalError (0.00s)
    encoding_test.go:265: got:
        
        wanted:
        error has been thrown
--- FAIL: TestError_simple (0.00s)
    export_test.go:28: got:  wanted: help me
```
These are all tests that attempt to marshal and unmarshal an error. It does not appear to matter whether I build with the bundled libsass or a system one.

--- a/encoding.go
+++ b/encoding.go
@@ -68,7 +68,12 @@ func unmarshal(arg SassValue, v interface{}) error {
 		return errors.New("Invalid SASS Value - Taylor Swift")
 	case reflect.String:
 		if libs.IsString(sv) || libs.IsError(sv) {
-			gc := libs.String(sv)
+			var gc string
+			if libs.IsString(sv) {
+				gc = libs.String(sv)
+			} else {
+				gc = libs.Error(sv)
+			}
 			//drop quotes
 			if t, err := strconv.Unquote(gc); err == nil {
 				gc = t

--- a/libs/encoding.go
+++ b/libs/encoding.go
@@ -181,6 +181,12 @@ func Len(usv UnionSassValue) int {
 	panic("call of len on unknown type")
 }
 
+func Error(usv UnionSassValue) string {
+	c := C.sass_error_get_message(usv)
+	gc := C.GoString(c)
+	return gc
+}
+
 func String(usv UnionSassValue) string {
 	c := C.sass_string_get_value(usv)
 	gc := C.GoString(c)

